endaq/calc/stats.py

```diff
@@ -46,21 +64,44 @@ def max_abs(
 
 
 def rms(
-    data: np.ndarray,
+    array: np.ndarray,
     axis: Union[None, typing.SupportsIndex, Sequence[typing.SupportsIndex]] = None,
     keepdims: bool = False,
 ):
```

Comment:
Type hint for return

Reply:
see 06f2b0ae993894190bbe6c5aa510cddf75c58bde